frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDashboard.tsx

```diff
@@ -53,6 +57,48 @@ const TeamEventCreditDashboard = (props: {
         : ''
     }.`;
 
+  const getHoursAttended = (attendance: TeamEventAttendance): number => {
+    const hours = attendance.hoursAttended;
+    if (hours !== undefined) return hours;
+    return 1;
+  };
+
+  const TecDetailsDisplay = (props: { attendanceList: TeamEventAttendance[] }): JSX.Element => {
+    const { attendanceList } = props;
+    return (
+      <Card.Group>
+        {attendanceList.map((attendance) => {
+          const teamEvent = allTEC.find((tec) => tec.uuid === attendance.eventUuid);
+          if (teamEvent !== undefined) {
+            return (
+              <Card>
+                <Card.Content>
+                  <Card.Header>{teamEvent.name} </Card.Header>
+                  <Card.Meta>{teamEvent.date}</Card.Meta>
+                  <Card.Meta>
+                    {`Number of Credits: ${
```

Comment:
Nitpick here: I might change the label to "Number of Credits Earned"? This makes more sense especially in the context of events with hours. I wonder if it could be misleading if say the event was 0.5 TEC/hour but it says "Number of Credits: 1" here.

And then you could call it "Number of Credits Requested" if it's pending.

@JacksonStaniec @henry-li-06 @kembayeb @oscarwang20 thoughts?

Reply:
My suggestion would be "Total credits." I dont think we need to include "requested" anywhere, since it is under the "Pending" section already

Reply:
What about "Credits from this Event" to make it more clear?

Reply:
I think "Total Credits" is good and won't be too long for the cards.